### ISSUE TITLE
Error: Cannot set property 'isHotTableEnv' of undefined

### ISSUE THREAD

**Issue by minalkumar2893** [handsontable/ngHandsontable#269]
After upgrading to latest version. When right clicking on the grid to show the context menu, i am getting this error:
![image](https://user-images.githubusercontent.com/46880667/94818850-303f9200-041c-11eb-9426-0a77a59feecf.png)

**Comment by AMBudnik** [comment]
Hi @minalkumar2893 

Can you share a demo where this issue can be replicable?

**Comment by minalkumar2893** [comment]
replicated the issue in this fiddle: https://jsfiddle.net/minalkumar2893/zsythx64/20/

**Comment by AMBudnik** [comment]
Thank you for sharing the demo. 

I have replicated the issue using Handsontable 8.0.0 and wrapper version 0.13.2, but also while loading minimal settings https://jsfiddle.net/r6gm2zfo/1/ using Handsontable v 2.0.0 and wrapper v 0.13.0

**Comment by minalkumar2893** [comment]
@AMBudnik We did some research on this issue, and we discovered that this property 'isHotTableEnv' is removed from version 8.0.0 onwards. Does it mean we should remove the same from ngHandsontable? We are not sure of the implications. or was this property renamed after version 8.0.0? is there something which we can do to help resolve the issue?

**Comment by AMBudnik** [comment]
You're right @minalkumar2893 this PR describes that the `isHotTableEnv` is removed https://github.com/handsontable/handsontable/pull/6625
I will confirm and come back to you.

**Comment by minalkumar2893** [comment]
@AMBudnik any updates?

**Comment by AMBudnik** [comment]
I am sorry but there are no ngHandsontable tasks for the upcoming roadmap. Nevertheless, I keep this ticket open in my system so I will inform you as soon as I get any update.

**Comment by elquimeras** [comment]
With the last update of library the hansontable lost scope when is used behind a ng-if, my solution was use a ng-show, maybe set a custom context to your grid api can solve the problem too if the first solution is not enought in your case.

Post your code to take a look around

**Comment by minalkumar2893** [comment]
This is how we use the nghandsontable directive:
We have a questionnaire with different response types, out of them one is table. We use the nghandsontable directive inside another custom directive for the table response type. and we include this directive only if there is a question with table response type. 
In our earlier code we had an ng-if inside custom directive, but even after removing it, i get the same error.

**Comment by minalkumar2893** [comment]
@elquimeras How do i set a custom context to my grid api?

**Comment by AMBudnik** [comment]
Hi @minalkumar2893 could you send me your requirements on mail support@handsontable.com? I'll do my best to help.
